guidelines/enhancement_template.md

```diff
@@ -265,6 +278,78 @@ enhancement:
 - Will any other components on the node change? For example, changes to CSI, CRI
   or CNI may require updating that component before the kubelet.
 
+### Impact of API Extensions
+
+Describe the API extensions here in detail, especially their impact on a cluster:
```

Comment:
this seems very focused on the case where a new api resource type is being added, but doesn't cover the case where someone is trying to extend an existing api with new fields or behavior.

how are "api extensions" defined?

1) new resource type?
2) new field on a resource type?
3) new webhook on an existing type (or being added as part of adding a new resource type)?

Reply:
actually i guess it's focused on the case where a new webhook or finalizer is being added?

Reply:
As stated above `API Extensions` is everything impacting APIs: webhooks, finalizers, CRDs. 
The effect of a new API field being added should be describe throughout the enhancement discussing that kind of extension of the existing API, no?